### PR TITLE
hw03-mongodb

### PR DESCRIPTION
**Pull request by stanislav-pravdiuk**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+DB_HOST=mongodb+srv://stanislav_pravdiuk:Sp23038888@db.byyndvr.mongodb.net/phonebook?retryWrites=true&w=majority
```
Comment (ceREAL133) [inline .env]: .env file shouldn't be visible on github its data must be confidential. Add this file to gitignore

File: .gitignore [hunk 2]
```diff
@@ -1,4 +1,6 @@
 node_modules/
-.env
+config.js
+env
```
Comment (ceREAL133) [inline .gitignore]: you forgot "." before "env"

File: models/contact.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+};
+
+const Contact = model('contact', contactSchema);
+
+module.exports = {
+    Contact,
+    schemas
+};
+// module.exports = {
+//     addSchema,
```
Comment (ceREAL133) [inline models/contact.js]: delete all useless comments

File: .gitignore (hunk 2 above)
Comment (stanislav-pravdiuk) [reply .gitignore]: ok

File: models/contact.js (hunk 3 above)
Comment (stanislav-pravdiuk) [reply models/contact.js]: ok

**Comment by stanislav-pravdiuk** [comment]
I have made all the necessary corrections.

**Review by ceREAL133 — APPROVED** [review]

**Pull request merged**